lib/models/Cat.js

```diff
@@ -0,0 +1,38 @@
+const pool = require('../utils/pool');
+
+module.exports = class Cat {
+  id;
+  name;
+  type;
+  url;
+  year;
+  lives;
+  is_sidekick;
+
+  constructor(row) {
+    this.id = row.id;
+    this.name = row.name;
+    this.type = row.type;
+    this.url = row.url;
+    this.year = row.year;
+    this.lives = row.lives;
+    this.is_sidekick = row.is_sidekick;
+  }
+  static async getAll() {
+    const { rows } = await pool.query('SELECT * FROM cat;');
+    return rows.map((row) => new Cat(row));
+  }
```

Comment:
For our get all function, we will only want to return the id and name of each cat. Remember, fixing this might cause your test for the /cats endpoint to fail.